etna/transforms/__init__.py

```diff
@@ -1,6 +1,7 @@
 from etna.transforms.base import PerSegmentWrapper
 from etna.transforms.base import Transform
 from etna.transforms.decomposition import BinsegTrendTransform
+from etna.transforms.decomposition import ChangePointsSegmentationTransform
```

Comment:
Add also `RupturesChangePointsModel`

Reply:
Sorry, my fault, this is not a transform, so it might be misleading that it can be imported via `from etna.transforms`, so remove it from init))